app/mailers/immediate_mailer.rb

```diff
@@ -5,12 +5,13 @@
 ###
 
 class ImmediateMailer < ApplicationMailer
-  def immediate(message, to)
+  def immediate(message, to, delivery_method_options = nil)
```

Comment:
maybe?
```suggestion
  def immediate(message, to, delivery_method_options = {})
```
or 
```suggestion
  def immediate(message, to, delivery_method_options = { delivery_method_options: nil })
```

Thank I don't think you need the assignment below.